components/combobox/events.md

```diff
@@ -212,54 +212,55 @@ You can use the `OnRead` event to provide data to the component according to som
 
 You can also call remote data through `async` operations.
 
+When using `OnRead`, make sure to set `TItem` and `TValue`.
+
 >caption Custom Data according to the user input in the ComboBox
 
 >tip You can also [debounce the service calls and implement minimum filter length]({%slug combo-kb-debounce-onread%}).
 
->important You should **change** **only** the `Data` of the ComboBox in the `OnRead` handler. You should **not** change other parameters such as `Value`, because this can lead to issues with the asynchronous nature of the event - the ComboBox cannot know whether the change of those parameters comes from somewhere external, and race conditions can occur with the arrival of the new data. Moreover, such a change is likely to be unwanted and unexpected for the end user and cause bad UX.
+>important You should **change** **only** the data of the ComboBox in the `OnRead` handler. You should **not** change other parameters such as `Value`, because this can lead to issues with the asynchronous nature of the event - the ComboBox cannot know whether the change of those parameters comes from somewhere external, and race conditions can occur with the arrival of the new data. Moreover, such a change is likely to be unwanted and unexpected for the end user and cause bad UX.
```

Comment:
Can we have that in a template or it is not possible to render a template in an important container?

Reply:
Done